testgres/node.py

```diff
@@ -1695,6 +1694,28 @@ def _get_bin_path(self, filename):
             bin_path = get_bin_path(filename)
         return bin_path
 
+    def _escape_config_value(value):
+        result = "'"
+
+        for ch in value:
+            if (ch == "'"):
+                result += "\\'"
+            elif (ch == "\n"):
+                result += "\\n"
+            elif (ch == "\r"):
+                result += "\\r"
+            elif (ch == "\t"):
+                result += "\\t"
+            elif (ch == "\b"):
+                result += "\\b"
+            elif (ch == "\\"):
+                result += "\\\\"
+            else:
+                result += ch
+
+        result += "'"
+        return result
```

Comment:
Для большей читаемости так еще можно
escape_map = {
        "'": "\\'",
        "\n": "\\n",
        "\r": "\\r",
        "\t": "\\t",
        "\b": "\\b",
        "\\": "\\\\"
    }
    return "'" + "".join(escape_map.get(ch, ch) for ch in value) + "'"

Reply:
I thought about using a map instead of these if-else, but I decided that it will be more complicated and wont allow me to add a logic to convert the special characters to their code representations :)